### PR TITLE
Fixes #24610: Improve tracing loggers

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/24610

* Simplify the log file name handling
* We don't need a non-blocking file logger

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: policies/rudder-cli/src/logs.rs [hunk 1]
```diff
@@ -52,111 +50,85 @@ impl FromStr for OutputFormat {
...
+            // .log extension
+            .filename_suffix("log")
+            .build(log_dir)
+            .context(format!(
+                "Failed to initialize rolling log file in '{}'",
+                log_dir.display()
+            ))?;
+        Some(
+            tracing_subscriber::fmt::layer()
+                .event_format(human_fmt)
```
Comment (Fdall) [inline policies/rudder-cli/src/logs.rs:99]: Shouldn't we add a timestamp for the logged lines in the file ?

Comment (amousset) [reply policies/rudder-cli/src/logs.rs:99]: _Bien vu_

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**